styleguide/derek/pattern-components/webinar-item/_webinar-item.scss

```diff
@@ -44,12 +44,22 @@
   font-weight: $font-weight-regular;
 }
 
-.rsWebinarTopic-item-link a {
-  color: $rackspace-red;
-  font-weight: $font-weight-regular;
-  text-decoration: underline;
+a {
+  .rsWebinarTopic-item-link {
+    color: $rackspace-red;
+    font-weight: $font-weight-regular;
+    text-decoration: underline;
+  }
 }
 
-.rsWebinarText-content p {
-  font-weight: $font-weight-regular;
+p {
+  .rsWebinarText-content {
+    font-weight: $font-weight-regular;
+  }
+}
+
+.rsTy-button {
+  background-color: $rackspace-red;
+  color: $white;
+  text-decoration: none;
 }
```

Comment:
This shouldn't be in the webinar css file.  It's not related to that.   Also, I think CTAs are a case where we can stick to non-maintainable classes since we tend to use those consistently across a variety of applications.   I think we can just stick with `<a href="" class="button lead">Text<a>` here.

Reply:
removing. I wasn't aware of the existing class and I didn't know where to otherwise add the css to.